lib/xen.pm

```diff
@@ -22,26 +22,30 @@ use utils;
 #   * autoyast profile
 #   * extra parameters for virsh create / xl create
 our %guests = (
-    A_sles12_PV => {
+    sles12PV => {
         autoyast     => 'autoyast_xen/xen-SLES12-SP3-PV.xml',
         extra_params => '--paravirt',
+        macaddress   => '52:54:00:78:73:a1',
         location     => 'http://mirror.suse.cz/install/SLP/SLE-12-SP3-Server-GM/x86_64/DVD1/',
     },
-    A_sles12_HVM => {
+    sles12HVM => {
         autoyast     => 'autoyast_xen/xen-SLES12-SP3-FV.xml',
         extra_params => '--hvm',
+        macaddress   => '52:54:00:78:73:a2',
         location     => 'http://mirror.suse.cz/install/SLP/SLE-12-SP3-Server-GM/x86_64/DVD1/',
     },
-    A_sles15_PV => {
-        autoyast     => 'autoyast_xen/xen-SLES15-PV.xml',
-        extra_params => '--paravirt',
-        location     => 'http://mirror.suse.cz/install/SLP/SLE-15-Installer-LATEST/x86_64/DVD1/',
-    },
-    A_sles15_HVM => {
-        autoyast     => 'autoyast_xen/xen-SLES15-FV.xml',
-        extra_params => '--hvm',
-        location     => 'http://mirror.suse.cz/install/SLP/SLE-15-Installer-LATEST/x86_64/DVD1/',
-    },
+    #sles15PV => {
+    #autoyast     => 'autoyast_xen/xen-SLES15-PV.xml',
+    #extra_params => '--paravirt',
+    #macaddress   => '52:54:00:78:73:a3',
+    #location     => 'http://mirror.suse.cz/install/SLP/SLE-15-Installer-LATEST/x86_64/DVD1/',
+    #},
+    #sles15HVM => {
+    #autoyast     => 'autoyast_xen/xen-SLES15-FV.xml',
+    #extra_params => '--hvm',
+    #macaddress   => '52:54:00:78:73:a4',
+    #location     => 'http://mirror.suse.cz/install/SLP/SLE-15-Installer-LATEST/x86_64/DVD1/',
+    #},
```

Comment:
Are these comments really needed? Usually we don't like commenting code in test, as git is here to track these changes. So if it doesn't hurt, please uncomment them (as I think you want to add it).

Reply:
I'm aware of this. Perhaps we use those domain names to communicate with VMs which are installed during the test. But those all are in our network so they should be unique as well because of at the same time the test may be running on several workers. I will discuss this next Monday in the office.